mlir/include/mlir/Dialect/Affine/Transforms/Transforms.h

```diff
@@ -14,6 +14,7 @@
 #ifndef MLIR_DIALECT_AFFINE_TRANSFORMS_TRANSFORMS_H
 #define MLIR_DIALECT_AFFINE_TRANSFORMS_TRANSFORMS_H
 
+#include "mlir/Dialect/Affine/IR/AffineOps.h"
```

Comment:
Prefer forward-declaring classes that are only used in function declarations instead of including their full definitions. This tends to improve compile time significantly.